api/setw.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+from geopy.geocoders import Nominatim
+from .weather import weather_details
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import telegram
+    import telegram.ext
+
+conn = sqlite3.connect('/db/stats.db', check_same_thread=False)
+cur = conn.cursor()
+
+cur.execute('''CREATE TABLE IF NOT EXISTS weatherpref
+               (userid int PRIMARY KEY, location text)''')
```

Comment:
Wouldn't it be more efficient to store user's lat/lon from GeoPy? That's the data being sent to the weather API anyway. This way we make an additional call to GeoPy.